Sming/Components/spiffs/component.mk

```diff
@@ -15,8 +15,9 @@ $(COMPONENT_RULE)$(SPIFFY):
 ## Application
 
 # This controls filesystem generation, it doesn't actually disable SPIFFS support in the application
-CACHE_VARS			+= DISABLE_SPIFFS
+CONFIG_VARS			+= DISABLE_SPIFFS
 DISABLE_SPIFFS		?= 0
+APP_CFLAGS			+= -DDISABLE_SPIFFS=$(DISABLE_SPIFFS)
```

Comment:
Maybe we should deprecate DISABLE_SPIFFS and for consistency use ENABLE_SPIFFS.  ENABLE_SPIFFS by default should be 1.

Reply:
Maybe something like `GENERATE_SPIFFS_FILESYSTEM`? I should really get partition tables sorted because that will make this variable obsolete. May I suggest leaving it for now?

PS. LOL. I've just added `DISABLE_WIFI`. I did ruminate about whether it should be `ENABLE_WIFI` but I figured the default of OFF makes more sense.

Reply:
> May I suggest leaving it for now?

Ok, that's fine. 

> I've just added DISABLE_WIFI. I did ruminate about whether it should be ENABLE_WIFI but I figured the default of OFF makes more sense. 

Well ... well... how about ENABLE_WIFI=0 (the default of course will be 1) , same as ENABLE_MALLOC_COUNT=0.